### DISCUSSION TITLE
how to return last page in "native = True" model?

### DISCUSSION THREAD

**Discussion by Norwalker** [zauberzeug/nicegui#2433] [Q&A]
### Description

my app has a button which jump a new random page?
but in native = True model my app can't return last page?
for insance 
how to from two return one？
![one](https://github.com/zauberzeug/nicegui/assets/24812147/3e36de97-ee14-46da-a652-41a29ccaece5)
![tow](https://github.com/zauberzeug/nicegui/assets/24812147/e7c1d981-e0b9-4dfb-ad07-cfa0aee11cb4)

**Comment by falkoschindler** [comment]
Sorry, I don't understand your question. What are the screenshots supposed to show? Maybe you can give a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)?
